admin-dev/themes/new-theme/js/components/grid/extension/position-extension.js

```diff
@@ -31,141 +31,145 @@ const $ = window.$;
  * Class PositionExtension extends Grid with reorderable positions
  */
 export default class PositionExtension {
-    constructor() {
-        return {
-            extend: (grid) => this.extend(grid),
-        }
+  constructor() {
+    return {
+      extend: (grid) => this.extend(grid),
     }
+  }
 
-    /**
-     * Extend grid
-     *
-     * @param {Grid} grid
-     */
-    extend(grid) {
-        this.grid = grid;
-        this._addIdsToGridTableRows();
-        grid.getContainer().find('.js-grid-table').tableDnD({
-            onDragClass: 'position-row-while-drag',
-            dragHandle: '.js-drag-handle',
-            onDragStart: () => {
```

Comment:
we don't need onDragStart event?

Reply:
no it actually was left from a previous implementation, but `this.originalPositions` is no longer used so no need for this event